client/src/components/ProfileDetailPanel.js

```diff
@@ -1,43 +1,58 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { Avatar, Button, Paper, Typography } from '@material-ui/core';
+// import { createConversation } from '../api/messages'; //SOCKETS WHEN DONE
 
 const useStyles = makeStyles((theme) => ({
   paper: {
-    padding: theme.spacing(2),
+    width: '100%',
+    padding: theme.spacing(3),
     textAlign: 'center',
     [theme.breakpoints.up('md')]: {
-      width: '30%',
-      minHeight: '100vh'
+      height: '100vh'
     },
   },
   avatar: {
-    width: 60,
-    height: 60,
-    margin: '10px auto',
+    width: 80,
+    height: 80,
+    margin: '20px auto',
     [theme.breakpoints.up('md')]: {
-      margin: '30px auto',
+      margin: '25px auto 30px',
+      width: 100,
+      height: 100,
     },
   },
   button: {
-    margin: '10px 0px 10px 0px',
+    width: '80% !important',
```

Comment:
I haven't needed to use !important in this project, was something preventing you from setting width?

Reply:
Yeah -- it was the only thing that worked for me. Let me know if you have a better solution.